python/lib/solver.cc

```diff
@@ -268,12 +286,24 @@ WrapLBS(py::module& slv)
       return self.ComputeFissionRate(*phi_ptr);
     },
     R"(
-    ???
+    Computes the total fission rate.
 
     Parameters
     ----------
-    nature: {'old', 'new'}
-        ???
+    nature : {'old', 'new'}
```

Comment:
nature is a weird var name

Reply:
I agree. I'll change it to something more appropriate.

Reply:
Fixed.